contracts/fast/FastTokenFacet.sol

```diff
@@ -276,15 +230,14 @@ contract FastTokenFacet is AFastFacet, IERC20 {
   function allowance(address owner, address spender)
       public view override returns(uint256) {
     LibFastToken.Data storage s = LibFastToken.data();
-    // If the allowance being queried is from the zero address and the spender
-    // is a governor, we want to make sure that the spender has full rights over it.
-    if (owner == address(0)) {
-      if (!FastAccessFacet(address(this)).isGovernor(spender)) {
-        revert ICustomErrors.RequiresFastGovernorship(spender);
-      }
-      return s.balances[owner];
-    }
-    return s.allowances[owner][spender];
+    // If the allowance being queried is over the reserve, make it so that
+    // an issuer member would have complete coverage of it. Otherwise, make it zero.
+    if (owner == address(0))
+      return IHasMembers(LibFast.data().issuer).isMember(spender)
+        ? s.balances[owner]
+        : 0;
```

Comment:
Maybe instead of this comment... something like:

_"If the allowance being queried is owned by the reserve, and the `spender` is an Issuer member, the `spender` owns the full balance of the `owner`. If they are not an Issuer member then their allowance in `0`. Otherwise, the usual allowance applies."_